recipe/fix-multiple-pkgs.patch

```diff
@@ -0,0 +1,14 @@
+diff --git a/src/catkin_pkg/packages.py b/src/catkin_pkg/packages.py
+index aa7208b..ddbb435 100644
+--- a/src/catkin_pkg/packages.py
++++ b/src/catkin_pkg/packages.py
+@@ -60,7 +60,8 @@ def find_package_paths(basepath, exclude_paths=None, exclude_subspaces=False):
+     for dirpath, dirnames, filenames in os.walk(basepath, followlinks=True):
+         if set(dirnames + filenames) & {'AMENT_IGNORE', 'CATKIN_IGNORE', 'COLCON_IGNORE'} or \
+             os.path.realpath(dirpath) in real_exclude_paths or \
+-                (exclude_subspaces and '.catkin' in filenames):
++                (exclude_subspaces and '.catkin' in filenames) or \
++                os.path.relpath(dirpath, basepath).startswith('pkgs/'):
```

Comment:
I love it! 

One other way we could make this slightly more "secure" might be to use `os.path.join(sys.prefix, "pkgs")`?

The `sys.prefix` should point to the prefix where `conda` is installed and thus the `pkgs` dir might be.

Reply:
I don't think that'll be sufficient, will it? As we want to avoid any `pkgs` folder, not only the one of the current environment? But I might be wrong - this is clearly a hack and some edge cases might cause trouble, but it will in either way be much better than the current situation.

Reply:
Ok I think you're right! This is better.